websites/T/Tracker.gg/dist/metadata.json

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "OGMatrix#8818",
```

The author name does not include the discriminator
```suggestion
    "name": "OGMatrix",
```